colorscad.sh

```diff
@@ -3,6 +3,9 @@ INPUT=$1
 OUTPUT=$2
 PARALLEL_JOB_LIMIT=${3:-8}
 
+# Add GNU coreutils to the path for macOS users (`brew install coreutils`).
+PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"
```

Comment:
Can this be limited to OSX-only, maybe by wrapping it in something like ```if [ "$(uname)" = Darwin ]```?
It's probably very unlikely to have an effect on non-OSX systems, but who knows what strange things people may have in their /usr/local/ folder :slightly_smiling_face: